index.css

```diff
@@ -69,3 +69,216 @@
   font-size: 16px;
   line-height: 24px;
 }
+
+.achivement {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.recent {
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 40;
+  line-height: 52px;
+  color: #1c1a19;
+}
+
+.works {
+  display: grid;
+}
+
+.storie h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.storie p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.tools {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  padding: 0;
+  margin: 0;
+  margin-bottom: 10px;
+}
+
+.tools li {
+  list-style: none;
+  padding: 10px;
+  margin-right: 10px;
+  border: 1px solid black;
+}
+
+.button {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 3%;
+  text-align: center;
+  background-color: #ff6b00;
+  color: #fff;
+  border: 2px solid #ff6b00;
+}
```

Comment:
- [ ] Please add more padding to the buttons, this is how the buttons should look like: 😁😁


![image](https://user-images.githubusercontent.com/20567503/176929859-46a247eb-0a75-4885-b05d-7af5a1cf297b.png)

Reply:
This issue is well resolved! ✔️